TodoApi/Extensions/OpenTelemetryExtensions.cs

```diff
@@ -55,15 +55,18 @@ public static WebApplicationBuilder AddOpenTelemetry(this WebApplicationBuilder
             })
             .WithTracing(tracing =>
             {
+                //we need to use AlwaysSampler to record spans
+                //from Todo.Web.Server, because there it no opentelemetry
+                //instrumentation
                 tracing.SetResourceBuilder(resourceBuilder)
+                        .SetSampler(new AlwaysOnSampler())
```

Comment:
```suggestion
                // We need to use AlwaysSampler to record spans
                // from Todo.Web.Server, because there it no OpenTelemetry
                // instrumentation
                tracing.SetResourceBuilder(resourceBuilder)
                       .SetSampler(new AlwaysOnSampler())
```